engines/wintermute/base/gfx/opengl/x/loader_x.cpp

```diff
@@ -0,0 +1,216 @@
+/* ResidualVM - A 3D game interpreter
```

Comment:
The X-loader is not specific to the OpenGL-renderer, it would be just as applicable for a TinyGL or Vulkan-renderer.

Reply:
done